### ISSUE TITLE
Failed to setup the bundle: spawn bundle ENOENT

### ISSUE THREAD

**Issue by tombohub** [Shopify/vscode-ruby-lsp#504]
Vscode: 1.76.2
OS: Win10
rubyLSP: 0.2.3

Installed in VSCode and did not start,, this is the error:

![image](https://user-images.githubusercontent.com/3029412/228631538-da74188a-5e92-4ec4-933e-f85ca706465c.png)

Tried to troubleshoot, followed steps then this error:

![image](https://user-images.githubusercontent.com/3029412/228632614-8585d885-d0bf-4420-a08d-cca4207a31dd.png)

this is the folder tree:

![image](https://user-images.githubusercontent.com/3029412/228632732-8e590128-6be7-4466-970f-074cba83243e.png)

**Comment by saimageshvar** [comment]
Facing similar issue. Using VS Code attached with a container. The Ruby LSP Channel shows the following
```
>> Running `bundle install`...
>> `bundle install` exited with code -2
```

**Comment by Moonlight-Angel** [comment]
Having the exact same issue with the same output in the Ruby LSP channel with VSCode attached to a container (Dev Containers extension)

**Comment by tombohub** [comment]
> Did you add the `.ruby-version` file after the warning? Have you tried restarting? Is there anything in the output tab under the Ruby LSP channel?

I installed the extension then followed the troubleshoot. Right now I opened the same project which already has .ruby-version file and got the error: 
![image](https://user-images.githubusercontent.com/3029412/229548322-dcc63a63-0a87-4ff6-83af-4a1af0247016.png)

I restarted VScode, yes. 

Where do I see Ruby LSP channel?

output is empty:
![image](https://user-images.githubusercontent.com/3029412/229548615-3484f5e8-c8c4-43dd-9e70-a97e442d227a.png)


I added `chruby` part, now getting another error. Here is the whole screenshot:
![image](https://user-images.githubusercontent.com/3029412/229549275-c98bb61d-074c-4090-a6aa-b67ed164cedf.png)

**Comment by tombohub** [comment]
@vinistock  yes, it's empty. Reason is that I didn't know I need to put anything inside. I'm beginner and just wanted to check out ruby in vscode. Thanks

I have put version and now having this error:

```
Failed to activate chruby environment: Command failed: undefined -lic 'chruby "3.1.3" && ruby --disable-gems -rjson -e "printf(%{RUBY_ENV_ACTIVATE%sRUBY_ENV_ACTIVATE}, JSON.dump(ENV.to_h))"' 'undefined' is not recognized as an internal or external command, operable program or batch file.
``

screenshot:

![image](https://user-images.githubusercontent.com/3029412/229900773-7b8308cf-d080-42a8-a6fd-96b4b0ffdbe3.png)

**Comment by vinistock** [comment]
It looks like your `SHELL` environment variable is not set. But given that you're using Windows, I don't think `chruby` supports it, does it? I think it requires `bash` or `zsh` to be your shell.

 If you're using Windows, I believe #525 will help Windows users activate the right Ruby version more easily.

**Comment by guledali** [comment]
Where is the guide for people running ruby through docker

**Comment by tombohub** [comment]
> using Windows, I don't think `chruby` supports it, does it? I think it requires `bash` or `zsh` to be your shell.

I am not sure, I am really not familiar with ruby ecosystem. So none of these version managers magers support Windows:

![image](https://user-images.githubusercontent.com/3029412/229915867-21c97d48-5641-4ba0-8c9e-9a44e4a2d878.png)

In that case I have to wait #525

**Comment by andyw8** [comment]
Sidenote: There are multiple approaches to running Ruby on Windows. If you use WSL then you will can have access to all the standard tools:

https://learn.microsoft.com/en-us/windows/wsl/install

**Comment by janbiedermann** [comment]
https://github.com/Shopify/vscode-ruby-lsp/pull/546

**Comment by vinistock** [comment]
I just released [v0.2.5](https://github.com/Shopify/vscode-ruby-lsp/releases/tag/v0.2.5) which includes some relevant fixes for Windows. If you'd like to try it, you should set your version manager to `none`. Please report back if more issues come up.

**Comment by tombohub** [comment]
Hi, I've tried, but have this error, what does it mean?

![image](https://user-images.githubusercontent.com/3029412/234722826-1d38c6b7-38ed-4a4f-a62b-d2f64b90d8ca.png)

**Comment by vinistock** [comment]
Apparently, on Windows you'll need custom activation. It will be solved by #525.

**Comment by vinistock** [comment]
Version v0.3.0 includes the new custom activation. Please try the [instructions](https://github.com/Shopify/vscode-ruby-lsp#custom-activation) and report back if any problems persist.
